components/eam/tools/topo_tool/bin_to_cube/bin_to_cube.nl

```diff
@@ -0,0 +1,5 @@
+&binparams
+  raw_latlon_data_file = '/p/lustre2/zhang73/DATA/DEM/S5P_OPER_REF_DEM_15_00000000T000000_99999999T999999_20160111T104226.NCL_24-3.nc'
+  output_file = 'S5P_OPER_REF_DEM_15_24-3_cube12000.nc'
+  ncube=12000
```

Comment:
are we sure we want this to be the "default"?
Maybe it would be better to create a few different namelist files with the ncube value in the file name so it forces the user to make a decision on which resolution they need rather than blindly using whatever is set up in the single namelist file?

Reply:
These namelists are small enough we could just make them input (command line) arguments as well. I'd prefer that to namelists personally.

Reply:
yes it shouldn't be the default, i just picked the one for my test

please check the updates which use the command arguments

Reply:
Maybe I'm missing something, but I don't see any changes where command line argument support was added?

Reply:
this nl had been deleted since 03/21 commits, and the command line arguments are put in bin_to_cube.F90:

```
  iargc = command_argument_count()
  if (iargc /= 3) then
      print *, "Usage: <executable> raw_latlon_data_file output_file ncube"
      stop
  end if

  call get_command_argument(1, raw_latlon_data_file)
  call get_command_argument(2, output_file)
  call get_command_argument(3, string_ncube)
  read(string_ncube, *) ncube
```